containerbuddy/config.json

```diff
@@ -1,25 +0,0 @@
-{
```

Comment:
I opened https://github.com/10up/triton-wordpress/pull/1 to move files back up to the top level. I'd very much prefer not hiding the Dockerfile and Containerbuddy configs in a sub-folder.